pkg/defaults/defaults.go

```diff
@@ -41,6 +41,9 @@ const (
 	// RuntimePath is the default path to the runtime directory
 	RuntimePath = "/var/run/cilium"
 
+	// ProxySocketPath is the default path to the proxy socket directory
+	ProxySocketPath = "/var/run/proxy"
```

Comment:
@jrajahalme 

because the sockets directory needs to be mounted as a whole to the host via hostpath volume - so envoy & agent can communicate with each other.

this would clash with the bootstrapconfig which gets mounted via configmap volumemount.

Reply:
@jrajahalme 

changed the default proxy socket path to `/var/run/cilium/envoy/sockets` - by building on top of the runDir/stateDir.

the bootstrapconfig gets generated (embedded) or mounted (daemonset - from configmap) into `/var/run/cilium/envoy`.